modules/emulator/NAMESPACE

```diff
@@ -33,4 +31,6 @@ export(minimize.GP)
 export(mvjump)
 export(nderiv)
 export(p)
+export(plot.jump)
+export(plot.mvjump)
```

Comment:
ok, so for most of our functions we've been moving from using a dot to using an underscore in order to avoid confusion with S3 methods. But in this case I think jump and mvjump actually are objects and thus we DO want these to be S3